api/v1alpha1/nonadminbackup_types.go

```diff
@@ -105,6 +105,16 @@ type NonAdminBackupStatus struct {
 
 	Phase      NonAdminBackupPhase `json:"phase,omitempty"`
 	Conditions []metav1.Condition  `json:"conditions,omitempty"`
+
+	// +optional
+	QueueInfo *QueueInfo `json:"queueInfo,omitempty"`
+}
+
+// QueueInfo holds the queue position for a specific VeleroBackup.
+// It is used to determine how many backups are scheduled before the given VeleroBackup in the OADP namespace.
+// and counts only VeleroBackups that are still subject to be handled by OADP/Velero.
+type QueueInfo struct {
```

Comment:
Do we need a QueueInfo struct here ? Is the idea that we want to keep this flexible for future enhancements ? like actually giving the names of the backups that are ahead in the processing queue ?

Reply:
I was thinking more about stats like time to completion or size of backups ahead of current one, that's why created struct and not a single value to not limit ourselves. But can change to a single val.

Reply:
yeah makes sense, lets keep it is as struct.